flasksrc/quiz.py

```diff
@@ -14,6 +15,26 @@ class ToyNetQuizItem(Schema):
 class ToyNetQuizGetResp(Schema):
     items = fields.List(fields.Nested(ToyNetQuizItem))
 
+class ToyNetQuizScorePostReq(Schema):
+    quiz_id = fields.Int(required=True)
```

Comment:
@ajlacey @blazebissar is there an issue with setting `required=True` in combination with `use_kwargs`?

Reply:
It's usually the case that setting `required=True` causes validation errors for us even when the arguments are being supplied. This means that we need to do our own validation and are basically using these schemas for API documentation with Swagger.

Reply:
@sksmith1 let's remove the `required=True` though if the library were working as expected you would be expected to include it!